client-java/controller/src/main/java/org/evomaster/client/java/controller/opensearch/operations/ComparisonOperation.java

```diff
@@ -0,0 +1,19 @@
+package org.evomaster.client.java.controller.opensearch.operations;
+
+public abstract class ComparisonOperation<V> extends QueryOperation {
+    private final String fieldName;
+    private final V value;
+
+    ComparisonOperation(String fieldName, V value) {
```

Comment:
default-scope in Java is very tricky, and possibly counter-intuitive. as such, I usually avoid it. consider if a `protected` makes sense here